src/components/views/rooms/RoomTile.js

```diff
@@ -104,6 +109,18 @@ module.exports = createReactClass({
         });
     },
 
+    onJoinRule: function(ev) {
+        if (!ev || !ev.event ) return;
+        /* eslint-disable camelcase */
+        const { event: { type, room_id } } = ev;
+        if (type !== "m.room.join_rules") return;
+        if (room_id !== this.props.room.roomId) return;
+        if ( !event.content ) return;
+        const { event: { content: { join_rule } } } = ev;
```

Comment:
we tend to use the `getContent` accessor instead of accessing the underlying event, as in case we ever started supporting encrypting this state event the above would break